src/odemis/driver/andorshrk.py

```
@@ -2064,6 +2090,45 @@ def _doSetFlipper(self, flipper: int, pos: int, check: bool):
             logging.warning("Failed to update turret position, detector offset might be incorrect", exc_info=True)
         self._updatePosition()
 
+    def _doSetGoffsetAbs(self, target_offset, *, allow_grating_offset=True):
+        target_offset = int(round(target_offset)) # ensure that we get integers for steps
```

Ruff (0.15.2)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)